react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -82,9 +83,28 @@ const Map: React.FC<MapProps> = ({
     );
 
     React.useEffect(() => {
+        console.log("Use effect dispatch");
```

Comment:
This hook is not getting called when I do selection in the toolbar to hide a layer.